disk/tests/conftest.py

```diff
@@ -23,6 +25,11 @@ def psutil_mocks():
         yield
 
 
+@pytest.fixture(scope='session')
+def blkid_cache_file():
+    return os.path.join(os.path.dirname(__file__), 'fixtures', 'blkid_cache_file')
```

Comment:
Agree that fixture might also be appropriate here.

But for consistency, you can probably move this near `mock_blkid_output` in `mocks.py`, and maybe make it a function instead of a fixture.

Reply:
Done